<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="shopping_project.mapper.MemberMapperTest" tests="6" skipped="0" failures="1" errors="0" timestamp="2020-11-19T12:54:35" hostname="DESKTOP-DKQES9C" time="0.088">
  <properties/>
  <testcase name="test01ConfirmIDSuccess" classname="shopping_project.mapper.MemberMapperTest" time="0.004"/>
  <testcase name="test02ConfirmIDFail" classname="shopping_project.mapper.MemberMapperTest" time="0.004"/>
  <testcase name="test03GetMemberSuccess" classname="shopping_project.mapper.MemberMapperTest" time="0.004"/>
  <testcase name="test04GetMemberFail" classname="shopping_project.mapper.MemberMapperTest" time="0.002"/>
  <testcase name="test05InsertMember" classname="shopping_project.mapper.MemberMapperTest" time="0.066">
    <failure message="org.springframework.dao.DuplicateKeyException: &#13;&#10;### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SHOPPING.SYS_C007100) violated&#10;&#13;&#10;### The error may exist in file [C:\Users\1234\git\Spring_shoppingmall_project\build\resources\main\mappers\MemberMapper.xml]&#13;&#10;### The error may involve shopping_project.mapper.MemberMapper.insertMember-Inline&#13;&#10;### The error occurred while setting parameters&#13;&#10;### SQL: INSERT INTO MEMBER(ID, PWD, NAME, EMAIL, ZIP_NUM, ADDRESS, PHONE)   VALUES(?, ?, ?, ?, ?, ?, ?)&#13;&#10;### Cause: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SHOPPING.SYS_C007100) violated&#10;&#10;; ORA-00001: unique constraint (SHOPPING.SYS_C007100) violated&#10;; nested exception is java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SHOPPING.SYS_C007100) violated&#10;" type="org.springframework.dao.DuplicateKeyException">org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SHOPPING.SYS_C007100) violated

### The error may exist in file [C:\Users\1234\git\Spring_shoppingmall_project\build\resources\main\mappers\MemberMapper.xml]
### The error may involve shopping_project.mapper.MemberMapper.insertMember-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO MEMBER(ID, PWD, NAME, EMAIL, ZIP_NUM, ADDRESS, PHONE)   VALUES(?, ?, ?, ?, ?, ?, ?)
### Cause: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SHOPPING.SYS_C007100) violated

; ORA-00001: unique constraint (SHOPPING.SYS_C007100) violated
; nested exception is java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SHOPPING.SYS_C007100) violated

	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:247)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:72)
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:88)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:440)
	at com.sun.proxy.$Proxy42.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:271)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:62)
	at org.apache.ibatis.binding.MapperProxy$PlainMethodInvoker.invoke(MapperProxy.java:152)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:85)
	at com.sun.proxy.$Proxy49.insertMember(Unknown Source)
	at shopping_project.mapper.MemberMapperTest.test05InsertMember(MemberMapperTest.java:72)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:84)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:251)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:97)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:190)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:110)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:38)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:62)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:118)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:182)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:164)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:412)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SHOPPING.SYS_C007100) violated

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:509)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:461)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1104)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:553)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:269)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:655)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:270)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:91)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:970)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1205)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3666)
	at oracle.jdbc.driver.T4CPreparedStatement.executeInternal(T4CPreparedStatement.java:1426)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3778)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.execute(OraclePreparedStatementWrapper.java:1081)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.execute(ProxyPreparedStatement.java:44)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.execute(HikariProxyPreparedStatement.java)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:59)
	at com.sun.proxy.$Proxy47.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:47)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:76)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:197)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:184)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:426)
	... 63 more
Caused by: Error : 1, Position : 0, Sql = INSERT INTO MEMBER(ID, PWD, NAME, EMAIL, ZIP_NUM, ADDRESS, PHONE)
		VALUES(:1 , :2 , :3 , :4 , :5 , :6 , :7 ), OriginalSql = INSERT INTO MEMBER(ID, PWD, NAME, EMAIL, ZIP_NUM, ADDRESS, PHONE)
		VALUES(?, ?, ?, ?, ?, ?, ?), Error Msg = ORA-00001: unique constraint (SHOPPING.SYS_C007100) violated

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:513)
	... 96 more
</failure>
  </testcase>
  <testcase name="test06ListMember" classname="shopping_project.mapper.MemberMapperTest" time="0.003"/>
  <system-out><![CDATA[DEBUG [Test worker] - test01ConfirmIDSuccess()
DEBUG [Test worker] - ==>  Preparing: SELECT 1 FROM MEMBER WHERE ID = ?
DEBUG [Test worker] - ==> Parameters: one(String)
TRACE [Test worker] - <==    Columns: 1
TRACE [Test worker] - <==        Row: 1
DEBUG [Test worker] - <==      Total: 1

DEBUG [Test worker] - test02ConfirmIDFail()
DEBUG [Test worker] - ==>  Preparing: SELECT 1 FROM MEMBER WHERE ID = ?
DEBUG [Test worker] - ==> Parameters: oneNone(String)
DEBUG [Test worker] - <==      Total: 0

DEBUG [Test worker] - test03GetMemberSuccess()
DEBUG [Test worker] - ==>  Preparing: SELECT ID, PWD, NAME, EMAIL, ZIP_NUM, ADDRESS, PHONE FROM MEMBER WHERE ID = ?
DEBUG [Test worker] - ==> Parameters: one(String)
TRACE [Test worker] - <==    Columns: ID, PWD, NAME, EMAIL, ZIP_NUM, ADDRESS, PHONE
TRACE [Test worker] - <==        Row: one, 1111, 김나리, knari@test.co.kr, 133-110, 서울시성동구성수동1가 1번지21호, 017-777-7777 
DEBUG [Test worker] - <==      Total: 1

DEBUG [Test worker] - test04GetMemberFail()
DEBUG [Test worker] - ==>  Preparing: SELECT ID, PWD, NAME, EMAIL, ZIP_NUM, ADDRESS, PHONE FROM MEMBER WHERE ID = ?
DEBUG [Test worker] - ==> Parameters: oneNone(String)
DEBUG [Test worker] - <==      Total: 0

DEBUG [Test worker] - test05InsertMember()
DEBUG [Test worker] - ==>  Preparing: INSERT INTO MEMBER(ID, PWD, NAME, EMAIL, ZIP_NUM, ADDRESS, PHONE) VALUES(?, ?, ?, ?, ?, ?, ?)
DEBUG [Test worker] - ==> Parameters: three(String), 1111(String), 세번째(String), three@test.co.kr(String), 133-110(String), 서울시 성동구 성수동 1가 1번지 21호(String), 010-333-3333(String)

DEBUG [Test worker] - test06ListMember()
DEBUG [Test worker] - ==>  Preparing: SELECT ID,PWD,NAME,EMAIL,ZIP_NUM,ADDRESS,PHONE,LEAVE_YN,JOIN_DATE FROM MEMBER WHERE NAME LIKE ?
DEBUG [Test worker] - ==> Parameters: %째%(String)
TRACE [Test worker] - <==    Columns: ID, PWD, NAME, EMAIL, ZIP_NUM, ADDRESS, PHONE, LEAVE_YN, JOIN_DATE
TRACE [Test worker] - <==        Row: three, 1111, 세번째, three@test.co.kr, 133-110, 서울시 성동구 성수동 1가 1번지 21호, 010-333-3333 , y, 2020-11-19 21:49:05
DEBUG [Test worker] - <==      Total: 1

]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
